bindings/python/pinocchio/shortcuts.py

```diff
@@ -12,29 +12,55 @@
 
 
 def buildModelsFromUrdf(
+    filename, *args, **kwargs
+) -> Tuple[pin.Model, pin.GeometryModel, pin.GeometryModel]:
+    """Parse the URDF file given in input and return a Pinocchio Model followed by corresponding GeometryModels of types specified by geometry_types, in the same order as listed.
+    Arguments:
+        - filename - name of the urdf file to load
+        - package_dirs - where the meshes of the urdf are located. (default - None)
+        - root_joint - Joint at the base of the model (default - None)
+        - root_joint_name - Name for the root_joint (default - "root_joint")
+        - verbose - print information of parsing (default - False)
+        - meshLoader - object used to load meshes (default - hpp::fcl::MeshLoader)
+        - geometry_types - Which geometry model to load. Can be pin.GeometryType.COLLISION, pin.GeometryType.VISUAL or both. (default - [pin.GeometryType.COLLISION, pin.GeometryType.VISUAL])
+    Return:
+        Tuple of the models, in this order : model, collision model, and visual model.
+
+    Example:
+        model, collision_model, visual_model = buildModelsFromUrdf(filename, root_joint, verbose, meshLoader, geometry_types, root_joint_name="root_joint_name")
+
+    Remark: In the URDF format, a joint of type fixed can be defined. For efficiency reasons, it is treated as operational frame and not as a joint of the model.
+    """
+
+    arg_keys = ["package_dirs", "root_joint", "verbose", "meshLoader", "geometry_types"]
+
+    for i, arg in enumerate(args):
+        if i < len(arg_keys):
+            kwargs[arg_keys[i]] = arg
```

Comment:
You can use do something like this:

```python
for key, arg in zip(arg_keys, args):
    kwargs[key] = arg
```
When input lists size mismatch, zip create a list with the smallest size.